packages/shared/styles/components/organisms/SfHero.scss

```diff
@@ -2,61 +2,58 @@
 
 .sf-hero-item {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  flex-direction: var(--hero-item-flex-direction, row);
+  align-items: var(--hero-item-align-items, unset);
+  justify-content: var(--hero-item-justify-content, flex-start);
```

Comment:
are we need these properties to be variables?

Reply:
I refactored this component to position container in similiar way to SfBanner, so this variables are useful now. Thanks.